android/engine/src/main/java/org/smartregister/fhircore/engine/ui/questionnaire/QuestionnaireActivity.kt

```diff
@@ -93,117 +100,117 @@ open class QuestionnaireActivity : BaseMultiLanguageActivity(), View.OnClickList
 
   override fun onCreate(savedInstanceState: Bundle?) {
     super.onCreate(savedInstanceState)
+    requestedOrientation = ActivityInfo.SCREEN_ORIENTATION_PORTRAIT
```

Comment:
Any reason for locking the screen orientation to portrait?

Reply:
The questionnaire is a long form which does not fit the landscape. Just a slight slip of hand causes screen go landscape and the activity starts again. Th reason was to block the orientation toggle if it happens in field.